src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig

```diff
@@ -53,10 +53,12 @@
         </div>
     {% else %}
         {# sort collected logs in groups #}
-        {% set deprecation_logs, debug_logs, info_and_error_logs = [], [], [] %}
+        {% set deprecation_logs, debug_logs, info_and_error_logs, silenced_logs = [], [], [], [] %}
         {% for log in collector.logs %}
             {% if log.context.level is defined and log.context.type is defined and log.context.type in [constant('E_DEPRECATED'), constant('E_USER_DEPRECATED')] %}
                 {% set deprecation_logs = deprecation_logs|merge([log]) %}
+            {% elseif log.priorityName == 'DEBUG' and log.context.scream is defined and log.context.scream == true  %}
```

Comment:
the condition `log.priorityName == 'DEBUG'` can be removed as the log priority doen't change the fact that its a silenced error